content/docs/apps/managing-teammates.md

```diff
@@ -19,6 +19,22 @@ If you invited them with an agency email address and they're part of an agency t
 
 After a teammate gets a cloud.gov account, an Org Manager or Space Manager will need to give them roles in your orgs and spaces so they can collaborate with you. For details about how org and space roles and permissions work, see [Cloud Foundry roles and permissions](http://docs.cloudfoundry.org/concepts/roles.html#roles).
 
+Org Managers can assign roles using the dashboard or command line. Space Managers can assign roles using the command line.
```

Comment:
Let's link this instance of `dashboard` to https://dashboard.fr.cloud.gov too. (One would hope a confused reader would keep reading to the next line, but we can't assume.)